hrldas/HRLDAS_forcing/create_forcing.F

```diff
@@ -469,8 +478,10 @@ program create_forcing
            call process(date, WEASDcurrent, WEASDprev, WEASDpost, geo_em, expand_loop);
            where (WEASDcurrent%field < 0) WEASDcurrent%field = 0
            call output_databuffer(ncid, WEASDcurrent)
-
+           
+           if (forcing_type/="ERA5") then ! ERA5 doesn't have Canopy water for initial state
```

Comment:
@CharlesZheZhang will this still generate CANWATER variable for ERA5 hrldas_setup file? The CANWATER variable is required by hrldas to initialize? Setting CANWATER to all 0 for ERA5 would be better?

Reply:
Yes, this is a good point. Will set CANWATER to 0 for ERA5 in the if-statement.